go.mod

```diff
@@ -17,9 +16,9 @@ require (
 	github.com/gorilla/websocket v1.4.2
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/improbable-eng/grpc-web v0.14.1
-	github.com/ipfs/go-ipld-cbor v0.0.4
+	github.com/ipfs/go-ipld-cbor v0.0.5
 	github.com/miguelmota/go-ethereum-hdwallet v0.0.1
-	github.com/multiformats/go-multihash v0.0.14
+	github.com/multiformats/go-multihash v0.0.10
```

Comment:
is this intended?

Reply:
no, this happens when we do `make proto-gen`